lib/routers/profile/pil.js

```diff
@@ -10,7 +11,7 @@ const submit = action => (req, res, next) => {
   const params = {
     data: {
       ...req.body.data,
-      establishmentId: req.establishment.id,
+      establishmentId: action === 'transfer' ? req.body.data.establishment.to.id : req.establishment.id,
```

Comment:
Now that each action is a standalone router it might be cleaner to do this in a middleware in the `/transfer` route.

```
(req, res, next) => {
  req.establishment.id = req.body.data.establishment.to.id;
  next(); 
}
```